### ISSUE TITLE
Deploy to Kubernetes is missing configmap from-env-file option

### ISSUE THREAD

**Issue by ATGardner** [Microsoft/vsts-tasks#7802]
## Troubleshooting
Checkout how to troubleshoot failures and collect debug logs: https://docs.microsoft.com/en-us/vsts/build-release/actions/troubleshooting

## Environment
- Server - VSTS or TFS on-premises?
    - using VSTS
        Account name: harmonie-collage
        Team project name: Collage
        Build definition name: testing AKS
        Build number: #2512
        Release definition name: testing AKS deploy

- Agent - Hosted or Private: 
    - If using private agent, provide the OS of the machine running the agent and the agent version: 
        Agent running Ubuntu
        Agent version: 2.133.3

## Issue Description
When using the Deploy to Kubernetes task version 1.*, I tried updating the configmap from file. In this sample deployment, I have an `out.env` file which contains

```
  A=value1   
  B=value2   
  C=value3
```

When I used the standard way of doing this, I got a configmap with only a single key of out.env, and a value which contains all three lines:
![image](https://user-images.githubusercontent.com/538720/43033701-dd510484-8cd6-11e8-931b-41cb8c1bdacc.png)

Looking at the VSTS deploy logs, I see the command that is running is
`kubectl --kubeconfig /var/vsts/_temp/kubectlTask/1532158955636/config create configmap configmapname --from-file=out.env=/var/vsts/r5/a/output/drop/out.env`

But the command that I actually want is `kubectl --kubeconfig /var/vsts/_temp/kubectlTask/1532158955636/config create configmap configmapname --from-**env**-file=out.env=/var/vsts/r5/a/output/drop/out.env`

It seems like this should be another option in the task.

### Error logs

[Insert error from the logs here]

**Comment by Anumita** [comment]
@ATGardner , Thanks for your feedback! We don't support the --from-env-file option right now. You could use the commands section to run the --from-env-file command to get unblocked. The file path has to be specified as it is specified in the configmaps section. Hope this solves your issue.

**Comment by ATGardner** [comment]
Yeah, I was able to use a workaround with another "Deploy to Kubernetes" step and a "create" command. There is also another problem there (maybe I should open another issue?) - if such a configmap already exists it fails. I tried adding another step before with a "delete" command (similar to how you do with the "Force update configmap"), but this this one failed because the output value has to be "name", and can't be "json" or "yaml".
Should I create another issue for this? (Can't use a "delete" command)
Or maybe I was missing something there?

**Comment by Anumita** [comment]
We print the output in the format provided in the Output format field in the Advanced section of the task. By default the value is json. The output format field is editable so you can keep the field empty. The delete command will not have the -o json argument then.
